learningresources/tests/test_views.py

```diff
@@ -184,3 +187,25 @@ def test_invalid_repo_form(self):
         body = resp.content.decode("utf-8")
         self.assertTrue("This field is required." in body)
         self.assertTrue(Repository.objects.count() == 1)
+
+    def test_export_good(self):
+        """Get raw XML of something I should be allowed to see."""
+        url = reverse("export", args=(self.resource.id,))
+        resp = self.client.get(url, follow=True)
+        body = resp.content.decode("utf-8")
+        self.assertTrue(resp.status_code == HTTP_OK)
+        self.assertTrue(self.resource.content_xml in body)
+
+    def test_export_no_permission(self):
+        """Get raw XML of something I should not be allowed to see."""
+        url = reverse("export", args=(self.resource.id,))
+        self.logout()
+        self.login(self.USERNAME_NO_REPO)
+        resp = self.client.get(url, follow=True)
+        self.assertTrue(resp.status_code == NOT_FOUND)
```

Comment:
I'm not sold about the 404 in case the resource exists and the user does not have permissions: I still think it's a 403